spec/spec_helper.rb

```diff
@@ -11,6 +12,10 @@
 FIXTURES_PROJECT_PATH = File.join(File.dirname(__FILE__), 'fixtures/fixtures.xcodeproj')
 
 RSpec.configure do |config|
+  config.before(:suite) do
+    `xcodebuild -project #{FIXTURES_PROJECT_PATH} -scheme fixtures -configuration Debug test`
```

Comment:
We can just do this before the suite in rspec rather than doing it in the Rakefile before running rspec.